### PR TITLE
Feature Button

### PR DESCRIPTION
**Pull request by abhijitdotsharma**
Button Component includes 
- Standard Buttons
- Outlined Buttons
- Icon Button
- Link Button
- Floating Action Button
Temp-button-nouCL

### CODE REVIEW

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    color: var(--color-white);
+    padding: 0.5rem 1rem;
+
+}
+.btn-large{
+    padding: 1rem 4rem;
+}
+.btn:hover{
+    background-color: var(--color-black);
+    /* opacity: 0.7; */
```
Comment (Bhakti199) [inline components/button/button.css:23]: remove this extra comment

**Review by Bhakti199 — APPROVED** [review]
looks good to me, you can merge it.

**Comment by sushil-mod** [comment]
code looks nice , you can merge!

**Review by sushil-mod — APPROVED** [review]
code looks nice , you can merge!

**Pull request merged**